MovieQuiz/Presentation/AlertPresenter.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  AlertPresenter.swift
+//  MovieQuiz
+//
+//  Created by Fixed on 17.05.2023.
+//
+
+import UIKit
+
+protocol AlertPresenterProtocol {
+    func showAlert(model: AlertModel)
+}
+
+class AlertPresenter: AlertPresenterProtocol {
+    weak private var delegate: UIViewController?
+    
+    init(delegate: UIViewController?) {
+        self.delegate = delegate
+    }
+    
+    func showAlert(model: AlertModel) {
+        let alert = UIAlertController(title: model.title,
+                                      message: model.message,
+                                      preferredStyle: .alert)
+
+        let action = UIAlertAction(title: model.buttonText, style: .default) { _ in
+            model.complection() }
```

Comment:
Закрывающая либо на той же строке, что и открывающая, либо на том же уровне что и открывающий код (как у классов и функций)
```suggestion
        let action = UIAlertAction(title: model.buttonText, style: .default) { _ in
            model.complection() 
        }
```